pkg/cincinnati/cincinnati.go

```diff
@@ -165,14 +165,15 @@ func (c Client) GetUpdates(ctx context.Context, uri *url.URL, arch string, chann
 	return current, requested, updates, nil
 }
 
-func (c Client) CalculateUpgrades(ctx context.Context, uri *url.URL, arch, sourceChannel, targetChannel string, version semver.Version, reqVer semver.Version) (Update, Update, []Update, error) {
-	var requested Update
+// CalculateUpgrades fetches and calculates all the update payloads from the specified
+// upstream Cincinnati stack given the current and target version and channel
+func (c Client) CalculateUpgrades(ctx context.Context, uri *url.URL, arch, sourceChannel, targetChannel string, version semver.Version, reqVer semver.Version) (current Update, requested Update, upgrades []Update, err error) {
```

Comment:
To make the signature smaller
```suggestion
func (c Client) CalculateUpgrades(ctx context.Context, uri *url.URL, arch, sourceChannel, targetChannel string, version, reqVer semver.Version) (current, requested Update, upgrades []Update, err error) {
```